README.md

```diff
@@ -1 +1,18 @@
-# redacted
\ No newline at end of file
+# Redacted
+
+Uses:
+- Node 12.18.4
+- Express 4.17
+
+Struggles:
+- Updating to Big Sur
```

Comment:
Protip: Do not install a major update to your system before doing a coding challenge! Among other things, the Command Line Tools for Mac were made obsolete upon updating, which broke Homebrew. The errors were I was getting were not very robust and it took a lot of research before figuring out I needed to update the CLT.